src/fs.zig

```diff
@@ -121,9 +121,8 @@ pub fn serveFile(
 
     //Carrot Return after headers to tell clients where headers end, and body starts
     try stream.writeAll("\r\n");
-    try response.flush();
```

Comment:
This should've been `try response.buffered_writer.flush()`. That is my mistake.

This will flush all buffered data at once. Saving us a few syscalls.